config/notifiers.go

```diff
@@ -638,6 +657,16 @@ type OpsGenieConfigResponder struct {
 	Type string `yaml:"type,omitempty" json:"type,omitempty"`
 }
 
+type OpsGenieConfigVisibleTo struct {
+	// One of those 3 should be filled.
```

Comment:
Comment doesn't match the behavior in the code. The code allows for **at least one** to be filled, not **at most one**?